### PR TITLE
Add support for analyticalDashboards to catalog-export

### PR DESCRIPTION
**Pull request by Tulean**
JIRA: RAIL-2839

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [x] `check` passes
-   [x] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by Tulean** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2035/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3669/ : SUCCESS in 31m 55s

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2036/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/592/ : SUCCESS in 39m 40s

File: tools/catalog-export/src/loaders/tiger/tigerAnalyticalDashboards.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+/**
+ * Load analytical dashboards that are stored in workspace metadata so that their links can be included
+ * in the generated output for easy embedding access.
+ *
+ * @param _projectId - project id, ignored for now as tiger is single-workspace
+ * @param tigerClient - tiger client to use for communication
+ */
+export async function loadAnalyticalDashboards(
+    _projectId: string,
```
Comment (no23reason) [inline tools/catalog-export/src/loaders/tiger/tigerAnalyticalDashboards.ts:14]: Please do not use _ prefix with parameters you are using in the function.

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/652/ : SUCCESS in 37s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/913/ : SUCCESS in 27s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3674/ : SUCCESS in 19m 16s

**Pull request merged**